client/src/components/Filters.js

```diff
@@ -0,0 +1,142 @@
+import React, {useState} from "react";
+import { Dropdown, Form, Radio } from "semantic-ui-react";
+import "./css/filters.css";
+
+function Filters(props) {
+    const [value, setValue] = useState(null);
+    const handleChange = (event, {value}) => setValue(value);
```

Comment:
This may not be super important right now but it seems like this is only storing one value in the state, but at some point we'll want all 3 values in state

Reply:
well this state thing is just for the radio button selection but if you mean we need to store the other variables from the dropdown then sure

Reply:
That's valid but when it comes time to submit data to the backend it's easiest if it's all in state

But we can leave that for another time if you want, no worries